src/components/network/line-layer.js

```diff
@@ -828,6 +927,7 @@ LineLayer.defaultProps = {
     lineFullPath: true,
     lineParallelPath: true,
     labelSize: 16,
+    iconSize: 48,
```

Comment:
why 48 ? other places mention 24

Reply:
Icon original size is 24px and I want it to be displayed in 48px.
The 24 value in the getLineIcon function refers to the original size.